engine/crates/engine/src/model/type.rs

```diff
@@ -189,16 +167,14 @@ impl<'a> __Type<'a> {
         }
     }
 
-    async fn input_fields(&self, ctx: &ContextField<'_>) -> Option<Vec<__InputValue<'a>>> {
-        if let TypeDetail::Named(registry::MetaType::InputObject(input_object)) = &self.detail {
+    async fn input_fields(&self, _ctx: &ContextField<'_>) -> Option<Vec<__InputValue<'a>>> {
+        if let TypeDetail::Named(registry_v2::MetaType::InputObject(input_object)) = &self.detail {
             Some(
                 input_object
-                    .input_fields
-                    .values()
-                    .filter(|input_value| is_visible(ctx, &input_value.visible))
+                    .input_fields()
+                    // .filter(|input_value| is_visible(ctx, &input_value.visible))
```

Comment:
I guess you could have removed all those filters instead of commenting, but well a nitpick for a +2/-2k PR